traffic_ops/app/db/reencrypt/reencrypt.go

```diff
@@ -0,0 +1,406 @@
+package main
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import (
+	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/trafficvault/backends/postgres"
+
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+
+	"crypto/aes"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+const PROPERTIES_FILE = "./reencrypt.conf"
+
+func main() {
+	previousKeyLocation := flag.String("previousKey", "", "The file path for the previous base64 encoded AES key.")
+	newKeyLocation := flag.String("newKey", "", "The file path for the new base64 encoded AES key.")
+	flag.Parse()
+
+	if previousKeyLocation == nil || *previousKeyLocation == "" {
+		fmt.Println("previousKey flag is required.")
+		os.Exit(0)
+	}
+	if newKeyLocation == nil || *newKeyLocation == "" {
+		fmt.Println("newKey flag is required.")
+		os.Exit(0)
+	}
+
+	newKey, err := readKey(*newKeyLocation)
+	if err != nil {
+		fmt.Println("reading newKey: ", err.Error())
+		os.Exit(0)
+	}
+
+	previousKey, err := readKey(*previousKeyLocation)
+	if err != nil {
+		fmt.Println("reading previousKey: ", err.Error())
+		os.Exit(0)
+	}
+
+	dbConfBytes, err := ioutil.ReadFile(PROPERTIES_FILE)
+	if err != nil {
```

Comment:
I think we should consider making this location configurable via a command line argument